flow/order/list.yaml

```diff
@@ -1,57 +1,164 @@
 steps:
-  # In:
-  #  user.id (str) - id of current user passed in from page
-  # Out:
-  #  flow.order_id (str) - id of selected order
-  - type: component.order.list
-    user_id: (@ user.user_id )
+  - (choose filter)
+  - flow_set:
+      fulfillment_status: any
+      financial_status: any
+      order_status: any
+
+  - ask: Please choose the order filter
+    buttons:
+      - text: Fulfillment status
+        action:
+          jump: fulfillment status
+      - text: Financial status
+        action:
+          jump: financial status
+      - text: Order status
+        action:
+          jump: order status
+      - text: See all orders
+        result: any
+  - flow_set: order_status
+  - jump: orders query
+
+  - (fulfillment status)
+  - ask: Select the fulfillment status
+    buttons:
+      - text: shipped
+        result: shipped
+      - text: partial
+        result: partial
+      - text: unshipped
+        result: unshipped
+      - text: unfulfilled
+        result: unfulfilled
+      - text: Change filter
+        action:
+          jump: choose filter
+  - flow_set: fulfillment_status
+  - jump: order status
+
+  - (financial status)
+  - ask: Select the financial status
+    buttons:
```

Comment:
Use quick replies. Also this is pretty bad UX. It's not a how a user would think about their order status